test/modules/post/test/cmd_exec.rb

```diff
@@ -37,6 +57,29 @@ def test_cmd_exec
       output == test_string
     end
 
+    it 'should execute show_args_* executables and return the passed arguments' do
+      if session.platform.eql? 'windows'
```

Comment:
Is there a cleaner way to write this? 👀 

It's a bit unweldy to read at a quick glance to understand what it's doing with the nested complexity here

Reply:
Here's a potential golf: 
https://github.com/adfoster-r7/metasploit-framework/commit/adc25ccbc941eb299fdedaf1203bb344dea79a88